controller/src/main/java/org/jboss/as/controller/ResourceDefinition.java

```diff
@@ -89,6 +89,63 @@ default void registerCapabilities(final ManagementResourceRegistration resourceR
         // no op
     }
 
+    /**
+     * Register "additional" Galleon packages that must be installed in order
+     * for this Resource to function. NB: the packages need to be visible from the feature pack
+     * that contains the ResourceDefinition. It can't be any package from any feature pack.
+     * The purpose of providing this information is to
+     * make it available to the Galleon tooling that produces Galleon feature-specs,
+     * in order to allow the tooling to include the package information in the relevant
+     * spec.
+     * <p>
+     * A package is "additional" if it is not one of the "standard" packages that must be
+     * installed. The names of "standard" packages should not be registered. The "standard"
+     * packages are:
+     *
+     *  <ol>
+     *      <li>
+     *          The root package for the process type; i.e. the package that provides
+     *          the main module whose name is passed to JBoss Modules when the process
+     *          is launched.
+     *      </li>
+     *      <li>
+     *          The package that installs the module that provides the extension in
+     *          which the resource is defined.
+     *      </li>
+     *      <li>
+     *          Any package that is non-optionally directly required
```

Comment:
directly or transitively

Perhaps this is wrong in the text I wrote that this was based upon. :)

Reply:
No you are right. Fixed.